tests/ui/invalid-compile-flags/crate-type-flag.rs

```diff
@@ -24,25 +23,17 @@
 
 //@[cdylib] ignore-musl (cdylib is supported, but musl libc is statically linked by default)
 //@[cdylib] ignore-ferrocenecoretest (cdylibs are not supported)
-=======
 //@[dylib] needs-crate-type: dylib
```

Comment:
this new directive should eliminate the need to specify these
- `ignore-musl`
- `ignore-wasm`
- `ignore-ferrocenecoretest`